src/ReactiveUI.AndroidSupport/ControlFetcherMixin.cs

```diff
@@ -67,20 +68,21 @@ public static T GetControl<T>(this View This, [CallerMemberName]string propertyN
                 () => This.FindViewById(controlIds[propertyName.ToLowerInvariant()]).JavaCast<T>());
         }
 
-        /// <summary>
-        /// This should be called in the Fragement's OnCreateView, with the newly inflated layout
-        /// </summary>
-        /// <param name="This"></param>
-        /// <param name="inflatedView"></param>
-        public static void WireUpControls(this global::Android.Support.V4.App.Fragment This, View inflatedView)
+		/// <summary>
+		/// A helper method to automatically resolve properties in an <see cref="Android.Support.V4.App.Fragment"/> to their respective elements in the layout.
+		/// This should be called in the Fragement's OnCreateView, with the newly inflated layout
+		/// </summary>
+		/// <param name="This"></param>
+		/// <param name="inflatedView">The newly inflated <see cref="View"/> returned from <see cref="LayoutInflater.Inflate"/>.</param>
+		/// <param name="resolveMembers">The strategy used to resolve properties that either subclass <see cref="View"/>, have a <see cref="WireUpResourceAttribute"/> or have a <see cref="IgnoreResourceAttribute"/></param>
+		public static void WireUpControls(this global::Android.Support.V4.App.Fragment This, View inflatedView, ResolveStrategy resolveMembers = ResolveStrategy.Implicit)
```

Comment:
Might be worth adding some unit tests to the Android tests project just to make sure we don't have a regression in the future.